generator.js

```diff
@@ -1,92 +1,97 @@
 	}
 
 	// copy a file
 	if (fs.existsSync(newFilePath)) {
-		console.warn(`File ${newFilePath} already exists`)
+		console.warn(`File ${newFilePath} already exists`);
 	} else {
-		const newFile = fs.readFileSync(filePath)
+		const newFile = fs.readFileSync(filePath);
 		if (filePath.match(/.*\.(png|ttf)/)) {
-			fs.appendFileSync(newFilePath, newFile)
+			fs.appendFileSync(newFilePath, newFile);
 		} else if (filePath.match(/gitignore/)) {
 			// the gitignore file becomes an npmignore and we need to
 			// change it back when laying down the project
-			const gitignorePath = newFilePath.toString().replace('gitignore', '.gitignore')
-			fs.appendFileSync(gitignorePath, newFile)
+			const gitignorePath = newFilePath.toString().replace('gitignore', '.gitignore');
+			fs.appendFileSync(gitignorePath, newFile);
 		} else {
 			// if it's not a binary file, treat it as a template
-			const templateFile = newFile.toString().replace(/%TITLE%/g, appTitle)
-			fs.appendFileSync(newFilePath, templateFile)
+			const templateFile = newFile.toString().replace(/%TITLE%/g, appTitle);
+			fs.appendFileSync(newFilePath, templateFile);
 		}
 	}
-}
+};
 
 const init = async () => {
-	const filePath = path.join(__dirname, 'template')
-	const projectPath = path.join(process.cwd(), appTitle)
+	const filePath = path.join(__dirname, 'template');
+	const projectPath = path.join(process.cwd(), appTitle);
 
 	// laying down the project files
-	console.log(`Creating ${projectPath} `)
-	console.log('Copying over project files')
-	processFile('', filePath)
+	console.log(`Creating ${projectPath} `);
+	console.log('Copying over project files');
+	processFile('', filePath);
 
 	// making an initial commit for git
-	if (commander.commit) {
-		console.log('Initializing a git repository')
+	if (options.commit) {
+		console.log('Initializing a git repository');
 
 		try {
-			const simplegit = git(projectPath)
-			console.log('Making the initial commit')
-			await simplegit.init()
-			await simplegit.add('.')
-			await simplegit.commit('Initial commit from Tram-One Express')
-			console.log('Successfully created commit')
+			const simplegit = git(projectPath);
+			console.log('Making the initial commit');
+			await simplegit.init();
+			await simplegit.add('.');
+			await simplegit.commit('Initial commit from Tram-One Express');
+			console.log('Successfully created commit');
 		} catch (error) {
-			console.log('Failed to create commit')
+			console.log('Failed to create commit');
 		}
+	} else {
+		console.log('Skipping making an initial commit');
 	}
 
 	// installing node dependencies
-	if (commander.install) {
-		console.log('Installing NPM Depenedencies')
-		execSync('npm install', { cwd: projectPath, stdio: 'inherit' })
+	if (options.install) {
+		console.log('Installing NPM Depenedencies');
+		execSync('npm ci', { cwd: projectPath, stdio: 'inherit' });
+	} else {
+		console.log('Skipping installing NPM depenedencies, run `npm ci` to get all dependencies required');
```

Comment:
Same as the comment above, this was for debugging, but also feels like a good quality of life change.